nautobot/docs/user-guide/administration/installation/index.md

```diff
@@ -1,13 +1,16 @@
 # Installation
 
-Nautobot can be deployed via Docker (Compose or Kubernetes) or directly onto a supported Linux system:
+Whether you are getting started for the first time, looking to build a local lab or ready to build a production environment, there are a variety of options to find the right fit for your needs with using Nautobot. 
 
-* [Nautobot Docker images](../guides/docker.md) are available via [Docker Hub](https://hub.docker.com/r/networktocode/nautobot) & [GitHub Container Registry](https://github.com/nautobot/nautobot/pkgs/container/nautobot) for use within a containerized environment
-    * Install Nautobot via [Docker Compose](https://github.com/nautobot/nautobot-docker-compose)
-    * [Install Nautobot via Helm Charts](https://docs.nautobot.com/projects/helm-charts/en/stable/) for Kubernetes
-* [Install Nautobot](install_system.md) onto a [RHEL (Red Hat Enterprise Linux)](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) or [Ubuntu](https://ubuntu.com/) virtual machine.
+## Deployment Options
 
-For more information about the Docker tags, Docker configurations, or using container images for your own development environment, see [Nautobot in Docker](../guides/docker.md).
+**Are you checking out Nautobot for the first time and don't want to install anything quite yet?** 
+
+You should consider playing around in the [Nautobot Always-On Sandbox Environments](https://networktocode.com/nautobot/sandbox-environments/). There is sample data already prepopulated, no installation required, and several Nautobot Apps are pre-installed, with sample data. 
```

Comment:
Perhaps?

```suggestion
You can explore the [Nautobot Always-On Sandbox Environments](https://networktocode.com/nautobot/sandbox-environments/). There is sample data already prepopulated, no installation required, and several Nautobot Apps are pre-installed, with sample data. 
```

Reply:
much simpler and same meaning, works for me. I will add to my updated commit